src/main/java/com/example/algorithm/KingAlgorithm.java

```diff
@@ -8,13 +8,18 @@
 import com.example.settings.AlgorithmSettings;
 import javafx.beans.property.BooleanProperty;
 import javafx.beans.property.SimpleBooleanProperty;
+import lombok.Getter;
+import lombok.Setter;
 
 
 public class KingAlgorithm implements Algorithm{
     private int phase = 0;
     private int numberOfPhases;
     private MyGraph<Integer, Integer> graph;
     private AlgorithmPhase round = AlgorithmPhase.SEND;
+
+    @Getter
+    @Setter
```

Comment:
Setting algorithm attribute from outside could be harmful. Is this setter necessary?

Reply:
You're right. I removed both unused setters.